### PR TITLE
Update the-ide-git-button.md

### PR DESCRIPTION
**Pull request by mirnawong1**
Added the new feature of reclone repo in the ide with added video

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md [hunk 1]
```diff
@@ -41,3 +41,9 @@ Once you click "initialize your project" you'll see the following in your IDE. C
 </p>
 
 **Pull changes from master/main**: This option will be available if you are on any local branch that is behind the remote version of the base branch or the remote version of the branch that you're currently on.
+
+## Reclone Repo
+Reclone your repository directly from the dbt IDE and your browser. You can reset your repository back to a fresh clone from your remote, you will want to click on the bottom right hand side green 'Ready'text, then click on the 'Reclone Repo' selection.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md]: These changes bring over changes from [this PR](https://github.com/dbt-labs/docs.getdbt.com/pull/1117). 

Note that in the future we'll be able to write this in one place and refer to it form both of these places. Here's the [PR for tracking this](https://github.com/dbt-labs/docs.getdbt.com/issues/1126).
```suggestion
Reclone your repository directly from the dbt IDE and your browser. To reset your repository back to a fresh clone from your remote, click **Ready**, which is green text in the bottom-right, then click **Reclone Repo**.
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
Hi @mirnawong1 Thanks so much for writing this content! I suggested some changes and like [the other PR](https://github.com/dbt-labs/docs.getdbt.com/pull/1117) that adds Recloning a repo, the video is super tiny and needs to be uploaded to `/img/docs/dbt-cloud/`. The reason we upload the videos is that adding them via dragging and dropping them into the PR is stored by GitHub and they don't guarantee keeping it. Please let me know if you have any questions or if you need help.

File: website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md]: hi @runleonarun , i've added the updated video today via the terminal today so it should no longer be tiny (hopefully) - please let me know if anything else is needed!

terminal confirmation:
```
Your branch is up to date with 'origin/mirnawong1-patch-17'.

nothing to commit, working tree clean
mirnawong@mirna-wong dbt-cloud % 
```

**Comment by mirnawong1** [comment]
closing this pr as the beta ide document covers this and will cover in more detail

**Pull request closed without merging**